src/extensions/default/JavaScriptCodeHints/main.js

```diff
@@ -56,6 +56,9 @@ define(function (require, exports, module) {
         noHintsOnDot   = false, // preference setting to prevent hints on dot
         ignoreChange;           // can ignore next "change" event if true;
 
+    // Languages that support inline JavaScript
+    var _inlineScriptLanguages = ["html", "php"];
```

Comment:
@abose jsp/asp files will be highlighted as [html files](https://github.com/adobe/brackets/blob/master/src/language/languages.json#L47).